### PR TITLE
Updated with tutorial

### PR DESCRIPTION
**Pull request by im2nguyen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 # Zscaler Private Access Terraform & Consul Webinar - Zero Touch App Delivery with ZPA, Terraform & Consul
 
-- This repository will provision Zscaler Private Access App Connector, Consul & NGINX servers in AWS. Optionally, you can also deploy a Vault Cluster in order to store the API credentials, so consul can retrieve it to authenticate to the ZPA Cloud.
+This repository will provision Zscaler Private Access App Connector, Consul & NGINX servers in AWS. Optionally, you can also deploy a Vault Cluster in store the API credentials, so Consul can authenticate to the ZPA Cloud.
```
Comment (willguibr) [inline README.md:14]: **Change to:** This repository will provision Zscaler Private Access App Connector, Consul & NGINX servers in AWS. Optionally, you can also deploy a Vault Cluster to store your ZPA API credentials, so Consul can authenticate to the ZPA Cloud.

**Review by willguibr — CHANGES_REQUESTED** [review]
Change to: This repository will provision Zscaler Private Access App Connector, Consul & NGINX servers in AWS. Optionally, you can also deploy a Vault Cluster to store your ZPA API credentials, so Consul can authenticate to the ZPA Cloud.

File: README.md (hunk 1 above)
Comment (im2nguyen) [reply README.md:14]:
```suggestion
This repository will provision Zscaler Private Access App Connector, Consul & NGINX servers in AWS. Optionally, you can also deploy a Vault Cluster to store your ZPA API credentials, so Consul can authenticate to the ZPA Cloud.
```

File: README.md [hunk 2]
```diff
@@ -168,25 +461,225 @@ consul-terraform-sync start -config-file config.hcl
...
 
-The Consul-Terraform-Sync execution, will automatically, create a new application segment within the Zscaler Private Access portal, containing the FQDNs of each one of our web servers with its corresponding service ports
+The CTS execution will automatically create a new application segment within the Zscaler Private Access portal, containing the FQDNs of each one of the web servers with its corresponding service ports.
 
 ![ZPAUI](assets/zpaconsole.svg)
 
-# How to test?
+### Verify NIA automation workflow
+
+This scenario will trigger CTS by increasing the number of servers allocated to the autoscaling group. CTS will update the BIG-IP configuration to include the new NGINX instances created by increasing the autoscaling group capacity.
```
Comment (im2nguyen) [inline README.md:470]:
```suggestion
This scenario will trigger CTS by increasing the number of servers allocated to the autoscaling group. CTS will update the ZPA application segment configuration to include the new NGINX instances created by increasing the autoscaling group capacity.
```

File: README.md [hunk 3]
```diff
@@ -168,25 +461,225 @@ consul-terraform-sync start -config-file config.hcl
...
+    }
+Plan: 3 to add, 3 to change, 0 to destroy.
+
+## ...
+Apply complete! Resources: 3 added, 1 changed, 0 destroyed.
+```
+
+Once the changes are applied on AWS, Consul will show the new instances on the **Services** tab.
+
+CTS will detect the change from the Consul catalog and modify the BIG-IP configuration to reflect the new `web-server` instances.
```
Comment (im2nguyen) [inline README.md:514]:
```suggestion
CTS will detect the change from the Consul catalog and update the application segments to reflect the new `web` services.
```

**Pull request merged**